app/models/work.rb

```diff
@@ -0,0 +1,47 @@
+class Work < ApplicationRecord
+  validates :title, presence: true
+  validates :title, uniqueness: true
+  has_many :votes
+  has_many :users, through: :votes, dependent: :destroy
+
+  def self.spotlight
+    # max = 0
+    # spotlight = nil
+    # Work.all.each do |work|
+    #   unless work.votes.length < max
+    #     max = work.votes.length
+    #     spotlight = work
+    #   end
+    # end
+    # return spotlight
+    return Work.all.sort_by { |work| work.votes.length }.last
+  end
+
+  def self.top_ten_movies
+    movies = Work.where(category: "movie").sort_by { |work| work.votes.length }.reverse
```

Comment:
You have this same method defined three times. Could you leverage the category to DRY this code?